cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -437,127 +497,62 @@ def test_reconfigure_vm_vmware_mem_multiple():
-        tags: reconfigure
-        testSteps:
-            1. get new configured appliance ->add vmware provider
-            2. provision 2 new vms
-            3. power off 1 vm -> select both vms
-            4. configure-->reconfigure vm
-            5. increase/decrease counts
-            6. power on vm
-            7. check changes
-    """
-    pass
-
-
-@pytest.mark.manual
-@pytest.mark.tier(2)
-@pytest.mark.provider([VMwareProvider])
-def test_reconfigure_vm_vmware_cores_multiple():
-    """ Test changing the cpu cores of multiple vms at the same time.
-
-    Polarion:
-        assignee: nansari
-        initialEstimate: 1/6h
-        testtype: functional
-        startsin: 5.6
-        casecomponent: Infra
-        tags: reconfigure
-        setup:
-            1. get new configured appliance ->add vmware provider
-            2. provision 2 new vms
-        testSteps:
-            1. Hot increase
-            2. Hot Decrease
-            3. Cold Increase
-            4. Cold Decrease
-            5. Hot + Cold Increase
-            6. Hot + Cold Decrease
-        expectedResults:
-            1. Action should fail
-            2. Action should fail
-            3. Action should succeed
-            4. Action should succeed
-            5. Action should fail
-            6. Action should Error
-    """
-    pass
-
-
-@pytest.mark.manual
-@pytest.mark.tier(3)
-@pytest.mark.provider([VMwareProvider])
-def test_reconfigure_add_disk_cold():
-    """ Test adding 16th disk to test how a new scsi controller is handled.
-
-    Bugzilla:
-        1337310
-
-    Polarion:
-        assignee: nansari
-        initialEstimate: 1/6h
-        testtype: functional
-        startsin: 5.7
-        casecomponent: Infra
-        tags: reconfigure
-        testSteps:
-            1. get new configured appliance ->add vmware provider
-            2. provision a new vm with 15 disks
-            3. Add a new disk with CloudForms using the VM Reconfigure dialog
-            4. Check new SCSI controller in vm
-    """
-    pass
-
-
-@pytest.mark.manual
-@pytest.mark.tier(2)
-@pytest.mark.provider([VMwareProvider])
-def test_reconfigure_add_disk_cold_controller_sas():
     """
+    vms = create_vms_modscope
+    all_view = navigate_to(vms[0], "AllForProvider")
+    entities = [all_view.entities.get_entity(surf_pages=True, name=vm.name) for vm in vms]
+    [entity.ensure_checked() for entity in entities]
+    all_view.toolbar.configuration.item_select("Reconfigure Selected items")
+
+    reconfig_view = vms[0].create_view(InfraVmReconfigureView, wait="20s")
+    if config_type == "memory":
+        reconfig_view.memory.fill(True)
+        reconfig_view.mem_size_unit.select_by_visible_text("GB")
+        current_memory = int(reconfig_view.mem_size.value)
+        if increase:
+            new_memory = current_memory + 1
+        else:
+            if current_memory < 1:
+                pytest.skip("Cannot test when the value is less than the minimum required.")
```

Comment:
The skip message seem to say something different than what the code seem to be doing. I think it should read
"Cannot decrease memory bellow 1".